GameData/002_AdditionalConfigs/Knes.cfg

```diff
@@ -1,585 +1,204 @@
-// Hermes configs
-@PART[Knes_Hermes]:FOR[RP-0]
-{
-	MODULE
-	{
-		name = ModuleAvionics
-		massLimit = 30.0
-		interplanetary = False
-	}
-}
-
-@PART[Knes_Hermes]:NEEDS[FeatureScience,RP-0]:BEFORE[RP-0-Kerbalism] 
-{ 
-	%capsuleTier = Mature
```

Comment:
Kerbalism science experiments and storage aren't configured by the RO configs! Shouldn't have removed this!